lib/timetable-form/Form.tsx

```diff
@@ -84,6 +88,18 @@ export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
     reader.readAsText(file);
   }, []);
 
+  useEffect(() => {
+    const loadLpas = async () => {
+      try {
+        const lpas = await fetchLPAs();
+        setLPAs(lpas.entities.map((lpa) => lpa.name));
+      } catch (error) {
+        console.error(error instanceof Error ? error.message : error);
```

Comment:
Not for this PR but we need to start thinking more about our error handling and unhappy paths